app/models/card.rb

```diff
@@ -1,8 +1,9 @@
 class Card < ApplicationRecord
   attr_reader :picture_remote_url
   belongs_to :user
+  belongs_to :deck
 
-  validates :original_text, :translated_text, :review_date, presence: true
+  validates :original_text, :translated_text, :review_date, :user_id, :deck_id, presence: true
```

Comment:
you don't need relation card -> user because cards is related only with deck, decks related with users.